### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by regineheberlein**
adding some additional pointers for myself

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +25,12 @@ Methods, post-ASpace transformation tools, and reports to support common SC acti
         ```
         BRANCH=<name> cap staging deploy
         ```
+     NB: You may need to run this with `bundle exec`
+     
+     NB: You may also need to run `bundle install --path vendor/bundle` first
+   1. cd to current on server
```
Comment (carolyncole) [inline README.md:31]: Did we want to put the server name `lib-jobs-staging1` and `lib-jobs-prod1`

File: README.md [hunk 2]
```diff
@@ -23,3 +25,12 @@ Methods, post-ASpace transformation tools, and reports to support common SC acti
...
+      1. `bundle install --path vendor/bundle`
+      
+      NB: if it gets into a weird state (can't find a gem):
+      `bundle config set --local path 'vendor/cache'`
+   1. cd out and back into current as needed to get the latest current
```
Comment (carolyncole) [inline README.md:36]: Should we add an example run here?
```
bundle exec ruby my_script_name >> my_script.log 2>&1 &
tail -300f my_script.log
```

**Review by carolyncole — CHANGES_REQUESTED** [review]
@regineheberlein great idea to add these to the readme.  I'm suggesting adding a couple other things, but they do not need to be added if you do not find them valuable.

File: README.md (hunk 1 above)
Comment (regineheberlein) [reply README.md:31]: can't hurt, right? (do we keep server names ever private?)

File: README.md (hunk 2 above)
Comment (regineheberlein) [reply README.md:36]: great idea, thanks!

File: README.md (hunk 1 above)
Comment (carolyncole) [reply README.md:31]: No, we generally do not.

**Review by carolyncole — APPROVED** [review]

**Pull request merged**